core/include/cubos/gl/debug_render.hpp

```diff
@@ -0,0 +1,92 @@
+#ifndef CUBOS_DEBUG_RENDER_HPP
+#define CUBOS_DEBUG_RENDER_HPP
+
+#include <list>
+#include <mutex>
+
+#include <glm/glm.hpp>
+#include <glm/gtc/quaternion.hpp>
+
+#include <cubos/gl/render_device.hpp>
+
+namespace cubos
+{
+    class Debug
```

Comment:
Write docs for the class also, just something brief to describe what this does. I've also just noticed the name of the file and the class don't match up, shouldn't it be named `debug_renderer.hpp`? Or maybe just `debug.hpp` then since that's name of the class. Another option is renaming the class to `DebugRenderer`.